src/EditorFeatures/Core/EditorConfigSettings/Aggregator/SettingsAggregatorFactory.cs

```diff
@@ -4,20 +4,29 @@
 
 using System;
 using System.Composition;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
 
 namespace Microsoft.CodeAnalysis.Editor.EditorConfigSettings;
 
 [ExportWorkspaceServiceFactory(typeof(ISettingsAggregator), ServiceLayer.Default), Shared]
-internal class SettingsAggregatorFactory : IWorkspaceServiceFactory
+internal sealed class SettingsAggregatorFactory : IWorkspaceServiceFactory
 {
+    private readonly IThreadingContext _threadingContext;
+    private readonly IAsynchronousOperationListener _listener;
+
     [ImportingConstructor]
     [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-    public SettingsAggregatorFactory()
+    public SettingsAggregatorFactory(
```

Comment:
>public SettingsAggregatorFactory( [](http://example.com/codeflow?start=4&length=33)

nit: isn't your reflex to switch these to default ctor syntax?

Reply:
yes.  will do in that other mop up pr.